Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
```

Comment:
Ensure that the environment variables GITHUB_TOKEN and OPENAI_API_KEY used in 'dev_server' are not exposed in logs or error messages to prevent potential security leaks. Consider implementing a more secure way to handle sensitive information in the task definitions.